config/default/model.conf.php

```diff
@@ -1,7 +1,10 @@
 <?php
+
 use oat\taoResourceWorkflow\model\wfmodel\JsonWorkflow;
 
-return new JsonWorkflow(array(
+return new JsonWorkflow(
+    array(
```

Comment:
I think we have some possible missing indentations here and in other places :)

Reply:
i think so, on this case, only files affected by my phpcbf appeared on the cs check, but we will find more of these in the future :). As we agreed to not fix phpcs on non specific phpcs fix branches, i will leave it as it is :)

Reply:
It's good practice to fix CS when it's possible.
In this case, you can see, that the code wasn't perfect, but at least indentation was good, but after this automatic fix the indentation is broken.
I see 2 ways:
* Fix the indentation (+ replace `array()` with `[]`, if possible)
* Rollback changes for this file (not include this one in this PR), as this one is not required by PR + has broken CS

Reply:
added an extra indent + [] (im neither a friend of using array()). Thanks Andrei! :)